.github/workflows/docs.yml

```diff
@@ -0,0 +1,125 @@
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+name: Docs
```

Comment:
Please make this line first to be consistent with other workflows

Reply:
What would be the name of the workflow? Just lowercase `docs`?

Reply:
`Docs` is fine just put it in the first line of the workflow